code/modules/mob/living/simple_animal/hostile/human/nazi.dm

```diff
@@ -0,0 +1,464 @@
+/mob/living/simple_animal/hostile/humanoid/nazi
+	name = "Nazi"
+	desc = "Sieg Heil!"
+	icon = 'icons/mob/hostile_humanoid.dmi'
+	icon_state = "nazi"
+	icon_living = "nazi"
+	icon_dead = "nazi"
+	icon_gib = "syndicate_gib"
+	casingtype = null
+
+	speak = list("Ze healing is not as revarding as ze hurting.","Schweinhunds!","Can you feel ze Schadenfreude?","Ach, was ist los?")
+	speak_emote = list("says")
+	emote_hear = list("says")
+	emote_see = list("hums")
+	attacktext = "punches"
+	attack_sound = "punch"
+	speak_chance = 5
+
+	health = 100
+	maxHealth = 100
+
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+
+	corpse = /obj/effect/landmark/corpse/nazi
+
+	items_to_drop = list(
+		/obj/item/weapon/kitchen/utensil/knife/nazi,
+		)
+
+	faction = "nazi"
+
+	ranged = 1
+	retreat_distance = 7
+	minimum_distance = 4
+	projectiletype = /obj/item/projectile/bullet/midbullet2
+	projectilesound = 'sound/weapons/Gunshot.ogg'
+	casingtype = null
+
+	environment_smash = 0
+
+	var/ammo = 8
+	var/reloads = 1
+
+/mob/living/simple_animal/hostile/humanoid/nazi/Life()
+	..()
+	overlays.len = 0
+	if(stance != HOSTILE_STANCE_IDLE)
+		if(ranged)
+			overlays += icon('icons/mob/in-hand/right/items_righthand.dmi',"gun")
+			if(melee_damage_upper > 10)
+				overlays += icon('icons/mob/in-hand/left/items_lefthand.dmi',"knife")
+		else
+			overlays += icon('icons/mob/belt.dmi',"gun")
+			if (melee_damage_upper > 10)
+				overlays += icon('icons/mob/in-hand/right/items_righthand.dmi',"knife")
+
+/mob/living/simple_animal/hostile/humanoid/nazi/Die()
+	droploot()
+	..()
+
+/mob/living/simple_animal/hostile/humanoid/nazi/Shoot(var/atom/target, var/atom/start, var/mob/user, var/bullet = 0)
+	if(..())
+		if(type == /mob/living/simple_animal/hostile/humanoid/nazi)
```

Comment:
There's got to be a better way to do this

Also the Shoot() proc should return a value (this one doesn't)